CHANGELOG.md

```diff
@@ -38,11 +36,14 @@ It contains the following changes compared to the last public release [r1.3](#r1
 ### Changed
 All changes are to align with Commonalities 0.5.0-alpha.1
```

Comment:
Please refer to the PR which have introduced the changes, see other repositories as examples.